messages/fr.json

```
@@ -356,29 +356,362 @@
+    "HERO_SPECIAL_OFFER": "Cadeau spécial : 50% de réduction",
+    "HERO_CREATE_YOUR": "Transformez vos",
+    "HERO_DIRECTORY": "idées",
+    "HERO_IN_MINUTES": "en quelques minutes",
+    "HERO_SUBTITLE": "Fondation de développement web révolutionnaire, construite avec",
+    "HERO_AI": "IA",
+    "HERO_LISTINGS": "Contenu",
+    "HERO_PAYMENT": "Paiements",
+    "HERO_SUBTITLE_FEATURES": "Gestion de contenu, CMS, Blog, Authentification, Newsletter, SEO, Thèmes et plus encore.",
+    "HERO_CTA_START": "Obtenir ever.works",
+    "HERO_CTA_PREVIEW": "Voir la démo",
+    "HERO_SOCIAL_PROOF": "Plus de 260 créateurs développent plus rapidement avec ever.works",
```

Biome (2.1.2)

[error] The key HERO_SUBTITLE was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)